Resources/Help/analyses/mlregressionboosting.md

```diff
@@ -1,3 +1,79 @@
 Boosting Regression
 ==========================
 
+Boosting works by sequentially adding predictors to an decision tree ensemble, each one correcting its predecessor. Boosting tries to fit the new predictor to the residual errors made by the previous predictor.
+
+### Assumptions
+- The predictors consist of continuous, nominal, or ordinal variables.
+- The target is a continuous variable.
+
+### Input 
+-------
+#### Assignment Box 
+- Target: In this box, the variable that needs to be predicted should be entered. 
+- Predictors: In this box the variables that provide information about the target variable should be entered. 
+
+### Data Split Preferences
+#### Holdout Test Data
+- Sample *x*% of all data: Choose a percentage to randomly sample from your data to derive prediction error. Generates an internal indicator variable that indicates whether the observation is included (1) or excluded (0) from the test set.
+- Add generated indicator to data: Add the generated test set indicator from the option above to your data set. Requires a column name.
+- Test set indicator: Use an indicator variable to select data for the test set.
+
+#### Training and Validation Data
+- Sample *x*% for validation data: Randomly sample a percentage from the remaining data (after selecting the test set).
+- K-fold with *k* folds: Partition the remaining data in *k* parts.
+
+### Training Parameters 
+#### Algorithmic Settings
+- Shrinkage: a shrinkage parameter applied to each tree in the expansion. Also known as the learning rate or step-size reduction 0.001 to 0.1 usually work, but a smaller learning rate typically requires more trees.
+- Interaction depth: Integer specifying the maximum depth of each tree (i.e., the highest level of variable interactions allowed. A value of 1 implies an additive model, a value of 2 implies a model with up to 2-way interactions, etc. Default is 1.
+- Min. observations in node: Integer specifying the minimum number of observations in the terminal nodes of the trees. Note that this is the actual number of observations, not the total weight.
+- Training data used per tree: select the percentage of training data that is used to train each individual tree.
+- Loss function: The loss function used. Can be either *Gaussian*, *Laplace*, or *t*.
+- Scale variables: scales the variables. Standardization ensures that values of variables from different scales range into a specific similar scale. As a result, standardizing provides numerical stability, which improves the clustering output. JASP uses the Z-score standardization of a mean of 0 and a standard deviation of 1. This option is selected by default.
+- Set seed: gives the option to set a seed for your analysis. Setting a seed will exclude random processes influencing an analysis.
```

Comment:
Would like to have this explained in more detail. For example, add what happens if you repeat analysis.